packages/core/package.json

```diff
@@ -75,7 +75,7 @@
   "sideEffects": false,
   "scripts": {
     "test": "yarn hardhat-test && yarn forge-test",
-    "forge-test": "./scripts/SetUpFoundryWithHardhat.sh && forge test",
```

Comment:
this breaks CI with `error Command "forge-test" not found`

Reply:
have addressed.